packages/estransform/lib/sourcemaps.js

```diff
@@ -17,6 +15,10 @@ const SOURCEMAP_REGEX = /(?:(\/\*+\s*?sourceMappingURL\s*=)([\s\S]*?)(\*\/))|(?:
  * @property {string} [file]
  */
 
+/**
+ * @typedef {import('@parcel/source-map').default} SourceMapNode
+ */
+
```

Comment:
for some reason it doesn't work without explicit typedef like that and I get

```
packages/estransform/lib/parser.js:126:53 - error TS2345: Argument of type '{ sources: readonly string[]; sourcesContent?: readonly (string | null)[] | undefined; names: readonly string[]; mappings: string; version: number; file?: string | undefined; sourceRoot?: string | undefined; } | SourceMap' is not assignable to parameter of type 'SourceMap'.
  Type '{ sources: readonly string[]; sourcesContent?: readonly (string | null)[] | undefined; names: readonly string[]; mappings: string; version: number; file?: string | undefined; sourceRoot?: string | undefined; }' is not assignable to type 'SourceMap'.
    Types of property 'sources' are incompatible.
      The type 'readonly string[]' is 'readonly' and cannot be assigned to the mutable type 'string[]'.

126                         code: inlineSourcemap(code, map),
```

I tried to play with types a bit, remove `@typedef {Object} SourceMap`, use original `VLQMap` which it looks to be and in other place use the class type itself, but it's a bit of a rabbit hole with new errors popping up on the go

maybe I just don't fully understand how you process source maps, but in any case I hope the removal of types from `packages/estransform/modules.d.ts` is already an improvement

Reply:
fixed by using https://github.com/chialab/rna/pull/157/files#r1314634375